### DISCUSSION TITLE
Country names format

### DISCUSSION THREAD

**Discussion by jmargutt** [rodekruis/ADA-API#14] [General]
@gulfaraz when creating an event, is there a standard format of country names that the API accepts? To make sure the position is correctly shown on the world map.

**Comment by gulfaraz** [comment]
@jmargutt In the interface, if the string exceeds the max-width, the excess text will be hidden with ellipses. The max-width depends on the browser resolution.

The API accepts a string of up to 255 characters length for the country parameter.

**Reply by jmargutt** [reply]
But how does the ADA Portal know where to place the country in the world map (first page of the ADA-UI)?

**Reply by gulfaraz** [reply]
There should be a co-ordinates (lat, lng) value in the event object. I have it in my local but it's not in production. I'll update it.
